docs/source/release_notes/release-notes.rst

```diff
@@ -83,6 +83,17 @@ develop
          - Reduced logging noise from large Cassandra gets and puts by removing ERROR messages and only providing stacktraces at DEBUG.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1590>`__)
 
+    *    - |new|
+         - Upon startup of a Cassandra-backed AtlasDB client with a
+           :ref:`Timelock block <timelock-client-configuration>`, the client will attempt to connect to the Timelock
+           Server and automatically fast-forward the Timelock Server's timestamp bound to that of the embedded service.
+           The client will now also *invalidate* the embedded service's bound, backing this up in a separate row in the
```

Comment:
We might want to consider combining the release notes for parts 1, 2 and 3. @rhero for SA and comment.